source/constructs/api/lambda/portal_config.py

```diff
@@ -7,56 +7,55 @@
 
 
 def __get_env_with_defaults(key, default_value): 
-	value = os.getenv(key)
-	if None == value or "" == value:
-		return default_value
-	else:
-		return value
+    value = os.getenv(key)
+    if None == value or "" == value:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)